src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.Apps/AllAppsPage.cs

```diff
@@ -4,11 +4,10 @@
 
 using System.Collections.Generic;
 using System.Linq;
-using AllApps.Programs;
 using Microsoft.CmdPal.Extensions;
 using Microsoft.CmdPal.Extensions.Helpers;
 
-namespace WindowsCommandPalette.BuiltinCommands.AllApps;
+namespace Microsoft.CmdPal.Ext.Apps.Programs;
```

Comment:
Did we want to have the project name align to the namespace? Should we just align on `...Ext.Programs`?